deploy-docs/README.md

```diff
@@ -25,10 +25,11 @@ jobs:
 
 ## Inputs
 
-| Name   | Required | Description                           |
-| ------ | -------- | ------------------------------------- |
-| token  | true     | The token for push to `gh-pages`.     |
-| latest | true     | Whether to create the `latest` alias. |
+| Name                | Required | Description                           |
+| ------------------- | -------- | ------------------------------------- |
+| token               | true     | The token for push to `gh-pages`.     |
+| latest              | true     | Whether to create the `latest` alias. |
+| mkdocs-requirements | false    | Optional requirements.txt for MkDocs  |
```

Comment:
- IMO, `mkdocs-requirements-txt` is clearer.
- `Optional` isn't necessary because it's explained by the `Required` value.

```suggestion
| mkdocs-requirements-txt | false    | `requirements.txt` for MkDocs  |
```